master/internal/rm/agentrm/agents.go

```diff
@@ -205,6 +205,7 @@ func (a *agents) createAgent(
 
 	var poolConfig *config.ResourcePoolConfig
 	for _, pc := range a.poolConfigs {
+		// The address of a loop variable is always the same. Use a temporary variable to capture the address.
```

Comment:
this is no longer the case in Go 1.22

https://go.dev/blog/loopvar-preview

Reply:
`make -C master check` complained when I removed `pc := pc`. I can remove the comment, though.